.github/copilot-instructions.md

```diff
@@ -0,0 +1,84 @@
+This is the codebase for a native port of the TypeScript compiler and language server.
+The source directories of interest that we have are:
+
+- `internal` - Contains the compiler and language server code.
+- `_extension` - Contains a preview VS Code extension code that integrates with the language server.
+
+Most of our development takes place in the `internal` directory, and most behaviors can be tested via compiler tests.
+
+Most development on the codebase is in Go.
+Standard Go commands and practices apply, but we primarily use a tool called `hereby` to build, run tests, and other tasks.
+Feel free to install `hereby` globally if it is easier, and run `hereby --list` to see all available commands.
+
+```sh
+hereby build  # Build the project
+hereby test   # Run tests
+hereby format # Format the code
+hereby lint   # Run linters
+```
+
+Always make sure code is formatted, linted, and tested before sending a pull request.
+
+## Compiler Features, Fixes, and Tests
+
+
+When fixing a bug or implementing a new feature, at least one minimal test case should always be added in advance to verify the fix.

```

Comment:
I don’t think this is true at this stage of development. Many things are adequately tested but have existing diffs that should be deleted. Even for fixes outside porting PRs, I would rather not add new tests that are wholly duplicative.